spec/views/events/show.html.erb_spec.rb

```diff
@@ -26,6 +26,14 @@
     expect(rendered).to have_text(I18n.t 'occupied_places', count: occupied_places, scope: [:events, :applicants_overview])
   end
 
+  it "displays counter" do
+    free_places = assign(:free_places, @event.compute_free_places)
+    occupied_places = assign(:occupied_places, @event.compute_occupied_places)
+    render
```

Comment:
This looks like a merging mistake to me, i think that test is duplicated now (if you view the whole file it is there twice)